simple-rest-server/get.go

```diff
@@ -3,12 +3,40 @@ package main
 import (
 	"fmt"
 	"net/http"
-
+    "os"
+	"encoding/csv"
 	"github.com/julienschmidt/httprouter"
 )
 
 func ListTask(respWriter http.ResponseWriter, request *http.Request, params httprouter.Params) {
-	fmt.Fprintf(respWriter, "Show Details for ToDo")
+	 csvfile, err := os.Open("tasks.csv")
+
+         if err != nil {
+                 fmt.Println(err)
+                 return
+         }
+
+         defer csvfile.Close()
+
+         reader := csv.NewReader(csvfile)
+
+         reader.FieldsPerRecord = -1 
```

Comment:
I think we should go with FieldsPerRecord = 0.

Negative means no error checking is done for number of fields for each record. 
0 means the number of fields is set by the first record
Positive means that this is the expected number of fields.

By setting to 0, we can more quickly catch errors.